src/Hng.Application/Features/UserManagement/Dtos/GoogleLoginRequestDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Features.UserManagement.Dtos
+{
+    public class GoogleLoginRequestDto
```

Comment:
does this dto map to any entity?

Reply:
no

Reply:
> no

ok. handle other feedbacks, and I will merge.